### PR TITLE
ECCI-511: Newsroom river of news looks like gov version.

### PR DESCRIPTION
**Pull request by siliconmeadow**
## Implementation & caveats
- Newsroom full twig template tweaked to have the page component
- News list view from .gov added
- Manual step may be required on deployments to edit the newsroom ID for the view

image

## This PR has been tested in the following browsers
- [x] Arc
- [ ] Edge
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] IE 11 (Windows)
- [ ] iOS Chrome
- [ ] iOS Safari
- [ ] Android Chrome
- [ ] Android Firefox
- [ ] Android default

### CODE REVIEW

File: themes/custom/essex/css/overrides.css [hunk 1]
```diff
@@ -1328,24 +1328,6 @@ tbody tr {
   padding: var(--spacing-larger) 0 var(--spacing);
 }
 
-
```
Comment (Polynya) [inline themes/custom/essex/css/overrides.css:1331]: Does this affect news article teaser in components?

File: themes/custom/essex/templates/content/node--localgov-newsroom--full.html.twig [hunk 2]
```diff
@@ -73,14 +73,14 @@
 
 {%
   set classes = [
```
Comment (Polynya) [inline themes/custom/essex/templates/content/node--localgov-newsroom--full.html.twig:75]: remove indent from this line to match the following lines

Comment (siliconmeadow) [reply themes/custom/essex/templates/content/node--localgov-newsroom--full.html.twig]: Fixed.

File: themes/custom/essex/css/overrides.css (hunk 1 above)
Comment (siliconmeadow) [reply themes/custom/essex/css/overrides.css:1331]: I think it's ok, what do you think @kurtisRogers-Nomensa:

image

**Review by Polynya — APPROVED** [review]

**Pull request merged**